Assets/FishFactory/Scripts/FactoryFishSpawner.cs

```diff
@@ -61,6 +69,21 @@ public class FactoryFishSpawner : MonoBehaviour
     )]
     private int tier2Percentage = 50;
 
+    [Header("Fish Gutting Settings")]
+    [SerializeField]
+    [Tooltip("If toggled, the fish will be assigned a state defining if it has been successfully gutted or not")]
+    private bool toggleFishGuttingChance;
+
+    [SerializeField]
+    [Range(0, 100)]
+    [Tooltip("The percentage of fish that should be successfully gutted")]
+    private int successfullGuttingChance = 65;
```

Comment:
should be 75% for a total 100%?

Reply:
there is a third state, last state is 10%